subprojects/core/src/main/java/org/gradle/caching/configuration/internal/DefaultBuildCacheConfiguration.java

```diff
@@ -87,11 +93,17 @@ public void local(Action<? super DirectoryBuildCache> configuration) {
         configuration.execute(local);
     }
 
+    @Nullable
```

Comment:
This should not be needed as it is inherited from `BuildCacheConfiguration`.

Reply:
Without it, our archunit checks for java bean nullability symmetry chokes.